Dockerfile

```diff
@@ -30,26 +33,23 @@ RUN zypper addrepo https://download.opensuse.org/repositories/isv:SUSE:Edge:Edge
 # 5. Network configurator
 RUN zypper install -y \
     xorriso squashfs  \
-    libguestfs kernel-default e2fsprogs parted gptfdisk btrfsprogs guestfs-tools lvm2 \
+    libguestfs kernel-default-base e2fsprogs parted gptfdisk btrfsprogs guestfs-tools lvm2 \
```

Comment:
Is the `kernel-default-base` package sufficient for the functionality? How does it differ from `kernel-base`?